docs/css/extra.css

```
@@ -20,17 +20,6 @@ div.admonition.block>.admonition-title {
     display: none;
 }
 
```

To bring back my admonitions :)

```suggestion

.admonition.new, details.new {
    color: var(--bs-success-text-emphasis);
    background-color: var(--bs-success-bg-subtle);
    border-color: var(--bs-success-border-subtle);
}
.admonition.example, details.example {
    color: var(--bs-info-text-emphasis);
    background-color: var(--bs-info-bg-subtle);
    border-color: var(--bs-info-border-subtle);
}

```

---

Applied this suggestion in https://github.com/mkdocs/mkdocs/commit/351b87939ddc61bc767e2b869e2a2673fe9af5ae